packages/cap-config-writer/src/cap-writer/pom-xml.ts

```diff
@@ -0,0 +1,93 @@
+import type { Editor } from 'mem-fs-editor';
+import type { Logger } from '@sap-ux/logger';
+import { xml2js, js2xml, type Element, type ElementCompact } from 'xml-js';
```

Comment:
would `fast-xml-parser` suffice in place of `xml-js` as we already use `fast-xml-parser` in this repo.

Reply:
we discussed this and decided its best to use `xml-js` for easy read and write of xmls. Thanks Austin.